.scripts/util.sh

```diff
@@ -59,25 +63,73 @@ is_generated() {
     fi
 }
 
+select_browser() {
+    local browser_engine="$1"
+
+    if [ "$browser_engine" = "Chromium" ]; then
+        BROWSER="Chromium"
+        BROWSER_INCLUDE_DIR="$CHROMIUM_INCLUDE_DIR"
+        BROWSER_LIB_DIR="$CHROMIUM_LIB_DIR"
+        SYS_OPTS="-DCHROMIUM_INCLUDE_DIR=$BROWSER_INCLUDE_DIR -DCHROMIUM_LIB_DIR=$BROWSER_LIB_DIR"
+    elif [ "$browser_engine" = "MSWebView2" ]; then
+        BROWSER="MSWebView2"
+        BROWSER_INCLUDE_DIR="$MSWEBVIEW_INCLUDE_DIR"
+        BROWSER_LIB_DIR="$MSWEBVIEW_LIB_DIR"
+        SYS_OPTS="-DMSWEBVIEW_INCLUDE_DIR=$BROWSER_INCLUDE_DIR -DMSWEBVIEW_LIB_DIR=$BROWSER_LIB_DIR"
+    else
+        throw_error "Unsupported browser engine option: $browser_engine. Program will terminate."
+        return 1
+    fi
+}
+is_browser_selected() {
+    if [ -z "$BROWSER" ]; then
```

Comment:
"Program will terminate" was maybe OK in the 70's, but we will try to be more user friendly in our troubled times :)

"A browser has not been selected. Please do `select_browser $BROWSER_OPTIONS`"

Users will be grateful, and good vibes about our product will spread.